test/lib/identity_code_test.rb

```diff
@@ -30,4 +30,30 @@ def test_is_valid_randomly_for_estonian_id_codes
     identity_code = IdentityCode.new('EE', array_of_codes.sample)
     assert(identity_code.valid?)
   end
+
+  def test_validates_estonian_id_codes_correctly
+
+    # Valid estonian ID code for person born in 19th century
+    identity_code = IdentityCode.new('EE', "14501234213")
+    assert(identity_code.valid?)
+
+    # Valid estonian ID code for person born in 20th century
+    identity_code = IdentityCode.new('EE', "38701234218")
+    assert(identity_code.valid?)
+
+    # Valid estonian ID code for person born in 21th century
+    identity_code = IdentityCode.new('EE', "60901234211")
+    assert(identity_code.valid?)
+
+    # Incorrect estonian ID codes
+    identity_code = IdentityCode.new('EE', "29511234213")
+    assert_not(identity_code.valid?)
```

Comment:
`assert identity_code.invalid?` ?

Reply:
identity_code doesn't have function `invalid?`. I wouldn't create this method because it doesn't solve anything nor provide functionality that's _really_ needed.

Reply:
You're right! I assumed that it's `ActiveModel`-based for some reason... I would still create `invalid?` with the following simple content:

```
def invalid?
  !valid?
end
```

since this is what we're testing: that a record is invalid. Negations are less readable, so if it can be avoided, I would avoid it.

Reply:
Okay, fine by me 😀 Adding your snippet and porting tests to use it as well.